### PR TITLE
LCFS - BUG - Early Issuance report title

### LINKED ISSUE #2928: LCFS - BUG - Early Issuance report title
**Describe the Bug**  
The Early Issuance report title is displaying the incorrect quarter. Currently, it is showing "Q2" when it should still be displaying "Q1". The quarter title logic does not match the intended calendar-based thresholds.

**Expected Behaviour**  
The Early Issuance report title should dynamically reflect the correct quarter based on the current date:
- "Q1" from January 1 to June 30
- "Q2" from July 1 to September 30
- "Q3" from October 1 to Dec 31
- "Q4 - Year-end Compliance report" starting Jan 1

**Actual Behaviour**  
The report is currently displaying "Q2" prematurely, prior to July 1.

**Implications**  
- Users are misled about the current reporting period.  
- May cause confusion regarding the timing and submission expectations.  
- Undermines trust in system accuracy and reporting integrity.

**Steps To Reproduce**  
User/Role: Any user accessing the Early Issuance report  
1. Navigate to the Early Issuance report section before July 1.  
2. Observe that the title incorrectly displays "Q2".  
3. Compare the current system date to validate expected logic.

**Notes**  
- Update the title generation logic to match the intended quarterly cutoffs.  
- Confirm behavior across all quarterly transitions, including time zone handling if relevant.  
- Add tests to verify quarter boundaries are accurately enforced.

### PR DESCRIPTION
**Pull request by kevin-hashimoto**
- fix: early issuance report title ranges being wrong, closes #2928

### CODE REVIEW

File: frontend/src/views/ComplianceReports/EditViewComplianceReport.jsx [hunk 1]
```diff
@@ -170,14 +170,14 @@ export const EditViewComplianceReport = ({ isError, error }) => {
         submittedYear > parseInt(compliancePeriod)
       ) {
         quarter = 4
-      } else if (month >= 0 && month <= 2) {
-        quarter = 1 // Jan-Mar: Q1
-      } else if (month >= 3 && month <= 5) {
-        quarter = 2 // Apr-Jun: Q2
+      } else if (month >= 2 && month <= 5) {
+        quarter = 1 // Mar-Jun: Q1
```
[the remark this answers is not in the document]
Comment (kevin-hashimoto) [reply frontend/src/views/ComplianceReports/EditViewComplianceReport.jsx:174]: If the draft was created in jan/feb, would it not fall under Q4 according to the new date ranges?

"Q1" from March 1 to June 30
"Q2" from July 1 to September 30
"Q3" from October 1 to December 31
"Q4 / Year-end compliance report" starting January 1
